micronaut/src/main/java/com/gluonhq/ignite/micronaut/view/FXMLView.java

```diff
@@ -28,6 +28,8 @@
 package com.gluonhq.ignite.micronaut.view;
 
 import com.gluonhq.ignite.micronaut.FXMLRootProvider;
+import com.gluonhq.ignite.micronaut.OnFXThread;
+import io.micronaut.aop.InterceptorBean;
 import javafx.beans.property.ReadOnlyObjectProperty;
```

Comment:
Two new imports seem to be unused here?

Reply:
Oops my bad, before commit I put the Interceptor annotation on the wrong class by mistake, and forgot to clean imports.